Panic in MarshalENV

```
panic: reflect: call of reflect.Value.Int on interface Value

goroutine 8 [running]:
reflect.Value.Int(...)
	/usr/local/go/src/reflect/value.go:1461
golift.io/cnfg.(*unparser).Member(0x918580?, {0x918580?, 0xc000428a50?, 0xc000446000?}, {0xc000442470, 0xe}, 0x50?)
	/go/pkg/mod/golift.io/cnfg@v0.2.1/unparse.go:154 +0xa9c
golift.io/cnfg.(*unparser).Anything(0xc0002b3948?, {0x918580?, 0xc000428a50?, 0x3?}, {0xc000442470, 0xe}, 0x0)
	/go/pkg/mod/golift.io/cnfg@v0.2.1/unparse.go:87 +0x1fc
golift.io/cnfg.(*unparser).Map(0x926d40?, {0x926d40?, 0xc0006385d0?, 0x1?}, {0xc000442454, 0x6}, 0x54?)
	/go/pkg/mod/golift.io/cnfg@v0.2.1/unparse.go:206 +0x234
golift.io/cnfg.(*unparser).Anything(0x9924e0?, {0x926d40?, 0xc0006385d0?, 0xa83888?}, {0xc000442454, 0x6}, 0x0)
	/go/pkg/mod/golift.io/cnfg@v0.2.1/unparse.go:85 +0x1b2
golift.io/cnfg.(*unparser).DeconStruct(0xc0002b3c78, {0x8e7c00?, 0xc0006385b0?, 0x16?}, {0x9b6a0e, 0x2})
	/go/pkg/mod/golift.io/cnfg@v0.2.1/unparse.go:51 +0x525
golift.io/cnfg.(*ENV).Marshal(0xc0000a95c8, {0x8e7c00?, 0xc0006385b0?})
	/go/pkg/mod/golift.io/cnfg@v0.2.1/env.go:62 +0x170
golift.io/cnfg.MarshalENV(...)
	/go/pkg/mod/golift.io/cnfg@v0.2.1/env.go:46
github.com/Unpackerr/unpackerr/pkg/unpackerr.(*Unpackerr).runCmdhook(0x0?, 0xc00017f0e0, 0x0?)
	/unpackerr/pkg/unpackerr/cmdhook.go:68 +0xd7
github.com/Unpackerr/unpackerr/pkg/unpackerr.(*Unpackerr).runCmdhookWithLog(0xc000123880, 0xc00017f0e0, 0xc0006385b0)
	/unpackerr/pkg/unpackerr/cmdhook.go:46 +0x5a
github.com/Unpackerr/unpackerr/pkg/unpackerr.(*Unpackerr).watchCmdAndWebhooks(0xc000123880)
	/unpackerr/pkg/unpackerr/start.go:176 +0x92
created by github.com/Unpackerr/unpackerr/pkg/unpackerr.Start
	/unpackerr/pkg/unpackerr/start.go:151 +0x42a
```

--- a/unparse.go
+++ b/unparse.go
@@ -137,10 +137,10 @@ func (p *unparser) Interface(field reflect.Value, tag string, _ bool) (Pairs, bo
 }
 
 // Member parses non-struct, non-slice struct-member types.
-func (p *unparser) Member(field reflect.Value, tag string, _ bool) (Pairs, error) {
+func (p *unparser) Member(field reflect.Value, tag string, _ bool) (Pairs, error) { //nolint:cyclop
 	output := Pairs{}
 
-	switch field.Interface().(type) {
+	switch val := field.Interface().(type) {
 	// Handle each member type appropriately (differently).
 	case error:
 		if err, _ := field.Interface().(error); err != nil {
@@ -148,14 +148,30 @@ func (p *unparser) Member(field reflect.Value, tag string, _ bool) (Pairs, error
 		}
 	case string:
 		output.Set(tag, field.String())
-	case uint, uint8, uint16, uint32, uint64:
-		output.Set(tag, strconv.FormatUint(field.Uint(), base10))
-	case int, int8, int16, int32, int64:
-		output.Set(tag, strconv.FormatInt(field.Int(), base10))
+	case uint:
+		output.Set(tag, strconv.FormatUint(uint64(val), base10))
+	case uint8:
+		output.Set(tag, strconv.FormatUint(uint64(val), base10))
+	case uint16:
+		output.Set(tag, strconv.FormatUint(uint64(val), base10))
+	case uint32:
+		output.Set(tag, strconv.FormatUint(uint64(val), base10))
+	case uint64:
+		output.Set(tag, strconv.FormatUint(val, base10))
+	case int:
+		output.Set(tag, strconv.FormatInt(int64(val), base10))
+	case int8:
+		output.Set(tag, strconv.FormatInt(int64(val), base10))
+	case int16:
+		output.Set(tag, strconv.FormatInt(int64(val), base10))
+	case int32:
+		output.Set(tag, strconv.FormatInt(int64(val), base10))
+	case int64:
+		output.Set(tag, strconv.FormatInt(val, base10))
 	case float64:
-		output.Set(tag, strconv.FormatFloat(field.Float(), 'f', -1, bits64))
+		output.Set(tag, strconv.FormatFloat(val, 'f', -1, bits64))
 	case float32:
-		output.Set(tag, strconv.FormatFloat(field.Float(), 'f', -1, bits32))
+		output.Set(tag, strconv.FormatFloat(float64(val), 'f', -1, bits32))
 	case time.Duration:
 		output.Set(tag, (time.Duration(field.Int()) * time.Nanosecond).String())
 	case bool:

